<launch>
    <node
      pkg="beginner_tutorials"
	  type="auctioneer2.py"
	  name="auctioneer"
	/>

	<node
	  pkg="beginner_tutorials"
	  type="bid_caller3.py"
	  name="bidder1"
	>
		<param name="robot_id" value="1" type="int"/>
		<param name="sleep" value="1" type="int"/>
		<param name="base_x" value="2.0" type="double"/>
		<param name="base_y" value="3.0" type="double"/>
	</node>
	<node
	pkg="beginner_tutorials"
	type="bid_caller3.py"
	name="bidder2"
	>
		<param name="robot_id" value="2" />
		<param name="sleep" value="2" type="int"/>
		<param name="base_x" value="4.0" />
		<param name="base_y" value="2.0" />
	</node>
	<node
	pkg="beginner_tutorials"
	type="bid_caller3.py"
	name="bidder3"
	>
		<param name="robot_id" value="3" />
		<param name="sleep" value="3" type="int"/>
		<param name="base_x" value="2.0" />
		<param name="base_y" value="2.0" />
	</node>
</launch>
